lab 8.py

```diff
@@ -0,0 +1,54 @@
+import os
+import hashlib
+from contextlib import contextmanager
+
+
+@contextmanager
+def hash_file(folder_path, block_size=65536):
+    try:
+        a_file = open(folder_path, 'rb')
```

Comment:
here you should have something like this:
```suggestion
       with open(folder_path, 'rb') as a_file:
           ...
```